spyder/app/tests/test_mainwindow.py

```diff
@@ -3760,46 +3759,49 @@ def test_ordering_lsp_requests_at_startup(main_window, qtbot):
 @flaky(max_runs=3)
 @pytest.mark.parametrize(
     'main_window',
-    [{'spy_config': ('main', 'show_tour_message', 2)}],
+    [{'spy_config': ('tours', 'show_tour_message', 2)}],
     indirect=True)
 def test_tour_message(main_window, qtbot):
     """Test that the tour message displays and sends users to the tour."""
     # Wait until window setup is finished, which is when the message appears
+    tours = main_window.get_plugin(Plugins.Tours)
+    tour_dialog = tours.get_container()._tour_dialog
+    animated_tour = tours.get_container()._tour_widget
     qtbot.waitSignal(main_window.sig_setup_finished, timeout=30000)
 
     # Check that tour is shown automatically and manually show it
-    assert CONF.get('main', 'show_tour_message')
-    main_window.show_tour_message(force=True)
+    assert CONF.get('tours', 'show_tour_message')
```

Comment:
```suggestion
    assert tours.get_conf('show_tour_message')
```

Now that `tours` is a plugin, we can access its config options using `get_conf/set_conf`.